gs-academy/README.md

```diff
@@ -0,0 +1,46 @@
+
+## Current Training Process
+
+The current training process consists of 2 types of trainings. 
+
+1. Ad-hoc - During onboarding or during projects, customers may be new to or wanting additional knowledge in a certain area of the cloud native stack. In this case we use the [training catalog](https://docs.giantswarm.io/general/training-catalog/) to create presentation style trainings with time for discussion and questions at the end. These trainings are created and given by the product teams that are responsible for and ultimately experts in their area of the cloud native stack. Example: Security, Observability, etc. Scheduling for this is typically initiated by the Account engineer and the date of training is up to availability of the customer and product team. 
+
+2. Standard - There is only one training currently that is "standard" which is the `Working with Giant Swarm` introduction. This training introduces support model, communication expectations, and high level view. This is typically done as part of onboarding or when customers have new joiners or teams added to the GS platform. Newly created and implemented was the second part of the kick-off training `Getting started with kubectl-gs` which is meant to learn about access and getting a deeper look into under the hood of the GS platform. Again, this is typically scheduled by the customer's Account Engineer. 
+
+## Proposal
+
+The idea would be to implement a formal program `Giant Swarm Academy` that would encompass onboarding, training, and continuing education for both new and current customers. The main points would consist of:
+
+- All ad-hoc and standard trainings would become regular, recurring trainings at desired intervals. Still to be discussed is the interval however the idea would be every team gives a their training once a month and the schedule is staggered. 
+
+```
+# Example
+First tuesday of every month - "Working with GS" and "Getting started with Kubectl GS"
+First thursday of every month - Kube 101
+Second tuesday of every month - Security 101
+Second thursday of every month - Gitops 101
+...
+```
+
+- All trainings should include an element of hands-on game or practical application of the concepts. Examples may be kube 101, everyone stands up a local minecraft server or security everyone rebuilds an image to be more secure. This encourages practice on the technical concept but also practice working with each other.  
+
+- We would build specifical certificates for linkedin and/or POAPs (proof of attendance protocol) style NFTs. This would enable people to show their achievements and allow the possibiliy of different knowledge levels based on the course such as `Foundations`, `Practioner`, and `Expert` as an example when moving on to more advanced sessions.  
+
+## Open Questions
+
+- Training schedule. What should the intervals be and what is the effort for the teams?
```

Comment:
We have to think if all of them would be online or if we use a platform we can combine an offline theory (some platforms are interactive) and an online part. This will help teams reuse content and not burn out

Reply:
It needs to be a side job and the certificates do not need to be provable.

Reply:
IMHO once a month is a lot of work, and to me this is similar to employee onboarding in a sense, I do not give my product intro once a month, but the latest recording is always available on demand, then once every 3 months roughly I give a new live one which then becomes the new recording. In some cases I make exceptions where I give the live one earlier (or later) based on high/low demand.